client/tailscale/localclient.go

```diff
@@ -807,11 +807,16 @@ func (lc *LocalClient) ExpandSNIName(ctx context.Context, name string) (fqdn str
 	return "", false
 }
 
-// Ping sends a ping of the provided type to the provided IP and waits
+// Ping sends a ping of the provided type and size to the provided IP and waits
 // for its response.
-func (lc *LocalClient) Ping(ctx context.Context, ip netip.Addr, pingtype tailcfg.PingType) (*ipnstate.PingResult, error) {
+//
+// For disco pings, the size argument specifies the length of the packet's payload, that
+// is, including the disco headers and message, but not including the IP and UDP headers.
+// If size is smaller than the minimum message size it's ignored.
+func (lc *LocalClient) Ping(ctx context.Context, ip netip.Addr, pingtype tailcfg.PingType, size int) (*ipnstate.PingResult, error) {
```

Comment:
we kinda sorta try to keep API stability in this package (one of the few in oss, as it's part of tsnet)

But OTOH this is weird enough that I don't think many people use this. You could GitHub/Google code search and see I suppose.

We might want a new method like we did for `StatusWithoutPeers`/`Status` and `BugReport`/`BugReportWithOpts`, etc.

Reply:
Yeah I wasn't sure how stable the API was myself. I think this is relatively easy to do, we can just make the old Ping() a wrapper around the new Ping()? Salman, what do you think?